doc/classes/Node3D.xml

```diff
@@ -139,91 +142,95 @@
 			<param index="2" name="up" type="Vector3" default="Vector3(0, 1, 0)" />
 			<param index="3" name="use_model_front" type="bool" default="false" />
 			<description>
-				Moves the node to the specified [param position], and then rotates the node to point toward the [param target] as per [method look_at]. Operations take place in global space.
+				Moves the node to the specified [param position], and then rotates the node to point toward the [param target] as per [method look_at]. This operation is calculated in global space (relative to the world).
 			</description>
 		</method>
 		<method name="orthonormalize">
 			<return type="void" />
 			<description>
-				Resets this node's transformations (like scale, skew and taper) preserving its rotation and translation by performing Gram-Schmidt orthonormalization on this node's [Transform3D].
+				Orthonormalizes this node's [member basis]. This resets this node's [member scale] to the default, but preserves the [member position] and [member rotation]. See also [method Transform3D.orthonormalized].
 			</description>
 		</method>
 		<method name="rotate">
 			<return type="void" />
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around axis, a unit [Vector3], by specified angle in radians.
+				Rotates this node's [member transform] around the [param axis] by the given [param angle], in radians. This operation is calculated in parent space (relative to the parent).
 			</description>
 		</method>
 		<method name="rotate_object_local">
 			<return type="void" />
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around axis, a unit [Vector3], by specified angle in radians. The rotation axis is in object-local coordinate system.
+				Rotates this node's local transformation around the [param axis] by the given [param angle], in radians. This operation is calculated in local space (relative to this node), and as such does not affect the [member position].
 			</description>
 		</method>
 		<method name="rotate_x">
 			<return type="void" />
 			<param index="0" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around the X axis by angle in radians.
+				Rotates this node's local transformation around its X axis by the given [param angle], in radians. This changes the pitch, without affecting the [member position].
```

Comment:
I checked the code and tested, it actually doesn't use the node's local axis, it's parent-relative.

```suggestion
				Rotates this node's local transformation around the parent's X axis by the given [param angle], in radians. This changes the pitch, without affecting the [member position].
```